release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -611,6 +611,34 @@ Previously, the `thinPoolConfig.overprovisionRatio` field in the `LVMCluster` cu
 [id="ocp-release-notes-web-console_{context}"]
 === Web console
 
+[id="ocp-4-18-administrator-perspective_{context}"]
+==== Administrator perspective
+
+This release introduces the following updates to the *Administrator* perspective of the web console:
+
+* A new setting for hiding the *Getting Started Resources* card on the *Overview* page allowing for maximum use of the dashboard.
```

Comment:
card name is`Getting started resources` 
<img width="1426" alt="Screenshot 2025-02-17 at 2 04 14 PM" src="https://github.com/user-attachments/assets/dcf7e618-5b5b-436c-b3ce-8e23a527b64a" />